### PR TITLE
Enhance initializeSystem hook description.

### PR DESCRIPTION
**Pull request by koertho**
Since I stumbled over the `initializeSystem` hook in the last time, I though I could add some information to the descriprion, to make it a little clearer what `before the request processing is started` means :)
What do you think?

### CODE REVIEW

File: docs/dev/reference/hooks/initializeSystem.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ aliases:
 
 
 The `initializeSystem` hook is triggered right after the system initialization
-process is finished and before the request processing is started.
+process is finished and before the request processing is started. This also means the request has no [contao request scope](https://docs.contao.org/dev/framework/routing/#request-scope) set and no language redirects have been executed.
```
Comment (fritzmg) [inline docs/dev/reference/hooks/initializeSystem.md]:
```suggestion
process is finished and before the request processing is started. This also means the request has no [contao request scope](/framework/routing/#request-scope) set and no language redirects have been executed.
```

**Review by fritzmg — CHANGES_REQUESTED** [review]
Yes, I am fine with that :). Just a nitpick.

File: docs/dev/reference/hooks/initializeSystem.md (hunk 1 above)
Comment (koertho) [reply docs/dev/reference/hooks/initializeSystem.md]: Makes sense :)

**Comment by fritzmg** [comment]
Thank you @koertho

**Pull request merged**